fieldservice/models/fsm_stage.py

```diff
@@ -45,8 +45,7 @@ def _default_team_ids(self):
     stage_type = fields.Selection([('order', 'Order'),
                                    ('equipment', 'Equipment'),
                                    ('location', 'Location'),
-                                   ('worker', 'Worker')], 'Type',
-                                  required=True)
+                                   ('worker', 'Worker')], 'Type', require=True)
```

Comment:
I'm afraid this is incorrect,  `required` is the correct field attribute.
Is the goal to remove the required attribute?